app/views/govuk_publishing_components/component_guide/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :title, "#{@component_doc.name} component" %>
+<%= render 'govuk_component/title', title: @component_doc.name, context: 'Component' %>
+
+<p class="component-description">
+  <%= @component_doc.description %>
+</p>
+
+<div class="component-show">
+  <div class="component-doc">
+    <h3>How to call this component</h3>
```

Comment:
(This view and the `fixture` view share a bit of code in common. I don't know if it's possible to commonise through a partial?)